Sources/Datadog/Datadog.swift

```diff
@@ -96,18 +123,29 @@ public class Datadog {
         )
     }
 
+    /// Sets the tracking consent regarding the data collection for the Datadog SDK.
+    /// - Parameter trackingConsent: new consent value, which will be applied for all data collected from now on
+    public static func set(trackingConsent: TrackingConsent) {
```

Comment:
sorry if we already discussed that, i might've forgotten
wouldn't it be handy to give the user a getter as well? it can be useful for debugging for example

Reply:
AFAIK, we've already discussed that with @xgouchet and @mariusc83 - we don't see any good use case for a getter. This setting is known on the user side and may be easily persisted there (even in memory). Adding it to the SDK costs additional threading consideration.

If `TrackingConsent` becomes problematic in debugging, we could rather include it in logs printed in `Debug.verbosity = .debug`. In any way, I'd rather wait for a real request / use case.

Reply:
if already discussed -> resolved